lib/rules/call-indentation.js

```diff
@@ -17,14 +18,91 @@ module.exports = function(context) {
    //--------------------------------------------------------------------------
 
    function validateCallIndentation(node) {
-      var lineIndentationMatches = helper.lineIndentationMatches(node.callee.loc.end.line, node.loc.end.line);
+      var lineIndentationMatches = helper.lineIndentationMatches(node.callee.loc.end.line, node.loc.end.line),
+          endParensLocation = node.loc.end,
+          lastArg = node.arguments[node.arguments.length - 1],
+          firstArg = node.arguments[0],
+          indentAmount = helper.indent,
+          calleeIndent = helper.lineIndent(node.callee.loc.start.line),
+          allArgsNewLine = node.loc.start.line !== firstArg.loc.start.line,
+          prevArg, nodesByStartLine;
 
+      nodesByStartLine = _.reduce(node.arguments, function(memo, arg) {
+         if (memo[arg.loc.start.line]) {
+            memo[arg.loc.start.line].push(arg);
+         } else {
+            memo[arg.loc.start.line] = [ arg ];
+         }
+         return memo;
+      }, {});
+
+      // check if closing parenthesis is correctly indented
       if (!lineIndentationMatches) {
          context.report({
             node: node,
             message: 'Call expressions must begin and end with the same indentation',
          });
       }
+
+      // check if first arg is on a new line
+      if (allArgsNewLine) {
+         _.each(node.arguments, function(arg) {
+            // check that all arguments start on their own line
+            if (nodesByStartLine[arg.loc.start.line].length > 1) {
+               context.report({
+                  node: node,
+                  loc: arg.loc.start,
+                  message: 'Argument should be on a new line',
```

Comment:
How about "When arguments are on their own line, they must all be on their own line."

one-line-per-argument looks strange to me.